scheduler/test/cook/test/simulator.clj

```diff
@@ -592,12 +609,15 @@
                                          :memory 10000
                                          :ncpus 3)))
         jobs (sort-by :submit-time-ms jobs)
-        hosts (for [i (range 120)]
-                (trace-host i 20000.0 20.0))
+        num-hosts 120
+        host-mem 20000.0
+        host-cpus 20.0
+        hosts (for [i (range num-hosts)]
+                (trace-host i host-mem host-cpus))
         cycle-step-ms 30000
-        config {:shares [{:user "default" :mem 2000.0 :cpus 2.0 :gpus 1.0}]
-               :scheduler-config {:rebalancer-config {:max-preemption 1.0}
-                                  :fenzo-config {:fenzo-max-jobs-considered 200}}}
+        config {:shares [{:user "default" :mem (/ host-mem 10) :cpus (/ host-cpus 10) :gpus 1.0}]
```

Comment:
super-ocd-level-nitpick: lexicographic ordering of keys in the nested maps.

Reply:
done